voicer-client/src/components/voice/Voice.jsx

```diff
@@ -7,6 +7,8 @@ import VoiceItem from "./VoiceItem";
 export default function Voice() {
   const [nameMatchesDB, setNameMatchesDB] = useState(true);
   const [data, setData] = useState([]);
+  const [voiceSearch, setVoiceSearch] = useState('')
+  const [searchTags, setSearchTags] = useState('')
```

Comment:
@alphaseinor Wouldn't tags be an array given there are multiple possible tags that can be added?

Reply:
this is so you can have a search bar that doesn't include things like ?tag0=this&tag1=is as you type in it. you need a change handler and something to parse the search query into the get request

Reply:
Makes sense. One thing we might do in a future release is make it so when a user presses space it auto-completes a tag (even if just styling on frontend) to make more obvious to the user that this is a tag search.